dts/bindings/rng/nxp,cssv2.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: CSSv2 (Crypto Subsystem)
+
+compatible: "nxp,cssv2"
```

Comment:
This name seems a little too generic.  Does `nxp,lpc-ccs-v2` make sense?

Reply:
Hi @galak ,
I asked for this change, because this cssv2 security subsystem is used on more NXP MCUs than LPC.  We want to avoid confusion to re-use this binding on non-LPC SOCs.

Thanks

Reply:
@DerekSnell ok, should at least be `nxp,css-v2`

Reply:
Thanks, @galak .  We can make that change